src/tests/ftest/server/metadata.py

```diff
@@ -397,12 +397,12 @@ def test_container_removal_after_der_nospace(self):
                         "##Expecting der_no_space RC: -1007, not seen")
                 else:
                     self.log.info(traceback.format_exc())
-                    self.log.info("(1.4)No space error shown with additional",
+                    self.log.info("(1.4)No space error shown with additional"
```

Comment:
I got the following:
(1.4)No space error shown with additional8 containers created, test passed.
You might want to add a space after "additional".